dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphHostControl.xaml.cs

```diff
@@ -159,19 +160,19 @@ public void UpdateGraph()
                 var yAxisTitle = "# Shares of Stock";
 
                 // If graph already exists update values. Otherwise, create new graph.
-                if (graphContainer.Children.Count > 0 && currentGraph != null)
+                if (_graphContainer.Children.Count > 0 && _currentGraph != null)
                 {
-                    currentGraph.UpdateGraphData(graphTitle, xAxisTitle, yAxisTitle, customer.Data);
+                    _currentGraph.UpdateGraphData(graphTitle, xAxisTitle, yAxisTitle, customer.Data);
                 }
                 else
                 {
-                    var graph = new BarGraph(compositor, compositionHost.hwndHost, graphTitle, xAxisTitle, yAxisTitle,
-                        (float)CompositionHostElement.ActualWidth, (float)CompositionHostElement.ActualHeight, currentDpiX, currentDpiY, customer.Data, windowRenderTarget,
+                    var graph = new BarGraph(_compositor, _compositionHost.hwndHost, graphTitle, xAxisTitle, yAxisTitle,
+                        (float)CompositionHostElement.ActualWidth, (float)CompositionHostElement.ActualHeight, _currentDpiX, _currentDpiY, customer.Data, _windowRenderTarget,
                         true, BarGraph.GraphBarStyle.PerBarLinearGradient,
-                        new List<Windows.UI.Color> { Windows.UI.Color.FromArgb(255, 246, 65, 108), Windows.UI.Color.FromArgb(255, 255, 246, 183) });
+                        new Windows.UI.Color[] { Windows.UI.Color.FromArgb(255, 246, 65, 108), Windows.UI.Color.FromArgb(255, 255, 246, 183) });
```

Comment:
The "Windows.UI.Color" is redundant. Should just be "new { Windows.UI.Color.... }

Reply:
updated